source/copilot-docs.txt

```diff
@@ -0,0 +1,69 @@
+.. _vsce-copilot-docs:
+
+==============
+/docs Command
+==============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The ``/docs`` command provides MongoDB-specific information,
+supplemented by links to MongoDB Documentation. The |copilot| is
```

Comment:
Could we change the second sentence to something like:

For the docs command, the underlying model of the MongoDB Copilot Extension leverages Retrieval-Augmented Generation (RAG) to ensure responses are based on the latest version of MongoDB Documentation.

Reply:
Done!